More Updates with hardware, updated automated calibration, other tweaks.

christhechris opened the pull request:
- many updates to cal code for robustness.
- tweaks to crop settings
- add more flags tfm setup
- add some plotting functions to plot raw data
- update cal file with new property for cal luminance, previously hard coded.
- fix nbs path issues from swap to notebooks in nbs folder.
- fix radiance conversion for mode 4+. The reference files were not being binned etc, this is now now done during tfm_setup. This also removes redundant work in the dn2rad function. The current structure does not work for mode 5 this way, which needs to be re thought (due to wavelength def being smaller then full array after smile correction).

YiweiMao commented:
Btw, I made a nbs folder because the home directory was getting cluttered. I also updated the DataCube class to save with coordinates (wavelength, cross-track, along-track) so it can be opened in QGIS and ENVI without modification. Shouldn't affect the calibration. I went on holidays so forgot to let you know earlier but I think you pulled my changes already.

christhechris commented:
Yup, got it sorted I think now, I’ll prob merge my stuff in over the next
couple days.

You prob saw I updated workflow setup to make py6s work using conda etc.

On Tue, 28 Dec 2021 at 15:16, Yiwei Mao ***@***.***> wrote:

> Btw, I made a nbs folder because the home directory was getting cluttered.
> I also updated the DataCube class to save with coordinates (wavelength,
> cross-track, along-track) so it can be opened in QGIS and ENVI without
> modification. Shouldn't affect the calibration. I went on holidays so
> forgot to let you know earlier but I think you pulled my changes already.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/openhsi/openhsi/pull/13#issuecomment-1001860200>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAF7CQ4IFPA6Q6BIREVGKZLUTE2YXANCNFSM5KZ4N4AQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were assigned.Message ID:
> ***@***.***>
>

christhechris commented:
I also made fixes related to nab folder, lots of paths needed updating etc.

On Tue, 28 Dec 2021 at 15:55, Chris Betters ***@***.***> wrote:

> Yup, got it sorted I think now, I’ll prob merge my stuff in over the next
> couple days.
>
> You prob saw I updated workflow setup to make py6s work using conda etc.
>
> On Tue, 28 Dec 2021 at 15:16, Yiwei Mao ***@***.***> wrote:
>
>> Btw, I made a nbs folder because the home directory was getting
>> cluttered. I also updated the DataCube class to save with coordinates
>> (wavelength, cross-track, along-track) so it can be opened in QGIS and ENVI
>> without modification. Shouldn't affect the calibration. I went on holidays
>> so forgot to let you know earlier but I think you pulled my changes already.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/openhsi/openhsi/pull/13#issuecomment-1001860200>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AAF7CQ4IFPA6Q6BIREVGKZLUTE2YXANCNFSM5KZ4N4AQ>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>> You are receiving this because you were assigned.Message ID:
>> ***@***.***>
>>
>
